discovery/discovery_test.go

```diff
@@ -350,20 +350,106 @@ func TestNewDBOrchestorPoolCache_PollOrchestratorInfo(t *testing.T) {
 		require.Contains(testOrchs, test)
 	}
 
-	cacheRefreshInterval = 1 * time.Second
+	cacheRefreshInterval = 1 * time.Millisecond
 	returnInfo = polledOrchInfo
-	pool.PollOrchestratorInfo(nil)
-	time.Sleep(3 * time.Second)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	pool.PollOrchestratorInfo(ctx, nil)
+	time.Sleep(5 * time.Millisecond)
 	dbOrchs, err = pool.store.SelectOrchs(nil)
 	require.Nil(err)
 	require.Len(dbOrchs, 3)
 	expPrice, _ = common.PriceToFixed(big.NewRat(1, 1))
 	for _, o := range dbOrchs {
-		t.Log(o)
 		assert.Equal(o.PricePerPixel, expPrice)
 	}
 }
 
+func TestNewDBOrchestorPoolCache_PollOrchestratorInfo_CallCount(t *testing.T) {
```

Comment:
The majority of this test is duplicated from `TestNewDBOrchestorPoolCache_PollOrchestratorInfo` - what do you think about just moving the call count check there?

Reply:
Yeah much better on second thought when incorporating the comment above